keras_tuner/engine/hypermodel.py

```diff
@@ -80,6 +80,22 @@ def _build_wrapper(self, hp, *args, **kwargs):
             hp = hp.copy()
         return self._build(hp, *args, **kwargs)
 
+    def fit(self, hp, model, *args, **kwargs):
+        """Train the model.
+
+        Args:
+            hp: HyperParameters.
+            model: `keras.Model` built in the `build()` function.
+            **kwargs: Anything the user defines. They are passed from
+                `Tuner.search()`. if contains `callbacks`, a list of prebuild
+                Keras callbacks for model checkpointing and tensorboard
+                configuration will be added to callbacks provided by the user.
```

Comment:
`callbacks` is always passed by `search()`, right? If the user passed a callbacks list, it gets appended to, and if no callbacks list is passed, one gets created?

Reply:
Yes, the description is revised.